doc/Language/syntax.pod6

```diff
@@ -223,109 +223,193 @@ say 'code again';
 
 =head2 X<Identifiers|identifier,identifiers>
 
-Identifiers are grammatical building blocks that occur in several kind
-of statements, giving a name to objects, data structures and blocks of
-code. These names must start with an alphabetic character (or an
-underscore), followed by zero or more word characters (alphabetic,
-underscore or number). You can also embed dashes C<-> or single quotes
-C<'> in the middle, but not two in a row, and only if followed
-immediately by an alphabetic character.
+Identifiers are grammatical building blocks that may be used to give a name
+to entities/objects such as constants, variables (e.g. Scalars) and routines
+(e.g. Subs and Methods). In a variable name, an identifier is normally preceded
+by a sigil; i.e. the sigil is not part of the identifier, only of the variable
+name.
```

Comment:
As long as you're making a precision here, it might be interesting to include sigilless containers and how they lose the \ when they are used.

Reply:
It's not a matter of documenting "dubious" practices. The problem is the use of the word normally. If you are focusing on identifiers, you can simply say that when there's a sigil, it's not a part of the identifier, without saying if that's the "normal" practice or not.

Reply:
OK, I see what you mean now. I suppose `In a L<variable name|/language/variables>, any sigil
(and twigil) precedes the identifier and does not form a part thereof.` would work?

Reply:
:+1: